app/src/main/java/org/fossasia/susi/ai/helper/Constant.java

```diff
@@ -22,6 +22,7 @@
 
     public static final String THEME = "theme";
     public static final String DARK = "dark";
+    public static final String LIGHT = "Light";
```

Comment:
Here "Light" has "L" in uppercase whereas "d" in "dark is in lowercase. Please keep it uniform.